conf/application.conf

```diff
@@ -28,6 +28,7 @@ play.filters.enabled += "play.filters.csp.CSPFilter"
 # ~~~~
 # Additional play modules can be added here
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
```

Comment:
Are we able to remove this if we are now using V2?

Reply:
@JakeGoodwin still needed for the time being as the DefaultAuthConnector requires it